src/machine.rs

```diff
@@ -271,12 +274,13 @@ impl<'tcx> Evaluator<'tcx> {
             panic_payload: None,
             time_anchor: Instant::now(),
             layouts,
+            stack: Default::default(),
```

Comment:
I'd prefer `Vec::default()` or `Vec::new()` as they give an indication about the type that is being instantiated.